code/modules/projectiles/updated_projectiles/guns/specialist.dm

```diff
@@ -799,52 +809,26 @@
 		current_mag.current_rounds++
 	return TRUE
 
-
-/obj/item/weapon/gun/launcher/rocket/reload(mob/user, obj/item/ammo_magazine/rocket)
```

Comment:
Can you reload the rocket gun? Have you tested this?

Reply:
Yes